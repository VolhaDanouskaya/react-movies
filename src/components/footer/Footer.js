import { Container } from '@material-ui/core';

import React from 'react';
import './footer.scss';

// eslint-disable-next-line import/no-absolute-path
import logo from '/public/images/logo.png';

const Footer = () => (
  <Container className="footer" fixed>
    <img
      className="logo-footer"
      src={logo}
      alt=""
      width="150px"
      height="20px"
    />
  </Container>
);

export default Footer;
