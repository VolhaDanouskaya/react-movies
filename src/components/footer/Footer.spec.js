import React from 'react';

import renderer from 'react-test-renderer';

import Footer from './Footer';

test('test link to home should match snapshot', () => {
  const page = renderer.create(<Footer />).toJSON;
  expect(page).toMatchSnapshot();
});
