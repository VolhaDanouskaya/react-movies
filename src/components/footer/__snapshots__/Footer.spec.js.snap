// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test link to home should match snapshot 1`] = `[Function]`;
